scripts/cmake/vcpkg_copy_tool_dependencies.cmake

```
@@ -19,7 +28,13 @@ This command should always be called by portfiles after they have finished rearr
 * [fltk](https://github.com/Microsoft/vcpkg/blob/master/ports/fltk/portfile.cmake)
 #]===]
 
-function(vcpkg_copy_tool_dependencies TOOL_DIR)
+function(vcpkg_copy_tool_dependencies)
+    cmake_parse_arguments(PARSE_ARGV 0 arg "" "TOOL_DIR" "DEPENDENCIES")
+    
+    if (NOT DEFINED arg_TOOL_DIR)
+        set(arg_TOOL_DIR "${CURRENT_PACKAGES_DIR}/tools/${PORT}")
+    endif()
+
```

This is a breaking change to the function's signature, which is not allowed.

```suggestion
function(vcpkg_copy_tool_dependencies TOOL_DIR)
    cmake_parse_arguments(PARSE_ARGV 1 arg "" "" "DEPENDENCIES")
```